<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buy_exchange</class>
 <widget class="QDialog" name="buy_exchange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>231</width>
     <height>107</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="count_lots">
      <property name="placeholderText">
       <string>Введите количество лотов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sum_lots">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Покупка на сумму</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="available_accs"/>
    </item>
    <item>
     <widget class="QPushButton" name="buy_but">
      <property name="text">
       <string>Подтвердить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
